config/imsim-config.yaml

```diff
@@ -198,6 +198,13 @@ stamp:
     world_pos:
         type: InstCatWorldPos
 
+    fft_photon_ops:
+        -
+            type: lsst_diffraction
+            telescope: LSST
+            band: { type: OpsimMeta, field: band }
+            latitude: -30.24463
```

Comment:
As mentioned on the Galsim PR, this photon op requires pupil sampling, so you need to include a pupil sampler op here too.  Probably also a time sampler.